ui/decredmaterial/collapsible.go

```diff
@@ -10,15 +10,34 @@ import (
 	"github.com/planetdecred/godcr/ui/values"
 )
 
+type IconStyle uint8
+
+const (
+	// Outline sets the outline material design chevron icon.
+	Chevron IconStyle = iota
+	// Fill sets the Fill chevron icon.
+	Caret
```

Comment:
these comments might have been referring to other variables, and the comment doesn't match the variable name